### DISCUSSION TITLE
DenseNet121 overfitting on I-SPY 1 3D data classification

### DISCUSSION THREAD

**Discussion by jamesben6688** [Project-MONAI/MONAI#6618] [Q&A]
I am using the DenseNet 121 (3D) to do the 3D MRI volume classification (pCR prediction) on the I-SPY 1 dataset ([https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=20643859](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=20643859)), by using the T2 post-contrast images. However the validation result is terrible (less than 70%). Moreover, the model will mostly output 0, I already used the weighted crossentropy.  

Have anyone done experiments on such data, are there anything special I should take care of?

**Comment by KumoLiu** [comment]
Hi @jamesben6688, if overfitting, perhaps you could try using a simpler network such as vgg. And you can also follow some data preparation introduced in https://github.com/MIC-DKFZ/nnUNet/tree/master such as how to do normalization.
Hope it can help you, thanks!
